src/span.py

```diff
@@ -39,6 +39,13 @@ class Span(BaseModel):
     end_index: int = Field(
         ..., gt=0, description="The end index of the span within the text"
     )
+
+    prediction_probability: Optional[float] = Field(
+        default=None,
```

Comment:
`None` or `1`?

Reply:
I changed to `None` after our chat as I figured it'd be good to be able to identify whether a model has tried to set confidence or not. this way we can explicitly disambiguate between predictions with probabilities and without

Reply:
fair!